src/Api/Auth/Controllers/WebAuthnController.cs

```diff
@@ -64,8 +69,24 @@ public async Task<WebAuthnCredentialCreateOptionsResponseModel> PostOptions([Fro
         };
     }
 
+    [HttpPost("assertion-options")]
+    public async Task<WebAuthnLoginAssertionOptionsResponseModel> AssertionOptions([FromBody] SecretVerificationRequestModel model)
+    {
+        var user = await VerifyUserAsync(model);
+        var options = _userService.StartWebAuthnLoginAssertion();
+
+        var tokenable = new WebAuthnLoginAssertionOptionsTokenable(WebAuthnLoginAssertionOptionsScope.PrfRegistration, options);
```

Comment:
Probably shouldn't just assume we're doing Prf Registration here

Reply:
No, I think you're right. There should be a difference between the states of the Token. `login != registration != update`. We can use the token to control for which state the request is in. This way we can ensure "single responsibility" for tokens.